DemoServer/AdditionalLanguages/java/src/main/java/net/ravendb/demo/staticIndexes/mapReduceIndex/metadata.json

```diff
@@ -0,0 +1,222 @@
+                },
+                {
+                    "Title": "Map-Reduce indexes via client code",
+                    "Type" : "link",
+                    "Url"  : "https://github.com/ravendb/book/blob/v4.0/Ch12/Ch12.md#defining-mapreduce-indexes-via-client-code"
+                }
+            ]
+        },
+        {
+            "Title": "Define the Map function",
+            "Slug": "step-3",
+            "DescriptionLines": [
+                "* Similar to the simple Map-Index, the `map` function defines what data is indexed per index-field.",
+                "",
+                "* The Map function is the first stage of the Map-Reduce index.<br>",
+                "  It indexes the specified fields _per document_ in the collection.",
+                "",
+                "    * _Country_ - per document, this index-field will contain the document's country data.",
+                "",
+                "    * _CountryCount_ - per document, the value of this field will be '1'.<br>",
+                "                   This field will be aggregated later, in the Reduce stage, accumulating the count from all Employees documents.<br>",
+                "                   The accumulative value of this field will represent the number of employees documents that contain the country."
+            ],
+            "Assets": [
+                {
+                    "Title": "Map Reduce Indexes",
+                    "Type" : "link",
+                    "Url"  : "https://ravendb.net/docs/article-page/4.2/java/indexes/map-reduce-indexes"
+                },
+                {
+                    "Title": "Map-Reduce and Aggregations",
+                    "Type" : "link",
+                    "Url"  : "https://github.com/ravendb/book/blob/v4.0/Ch11/Ch11.md#mapreduce-and-aggregations-in-ravendb"
+                },
+                {
+                    "Title": "Map-Reduce indexes via client code",
+                    "Type" : "link",
+                    "Url"  : "https://github.com/ravendb/book/blob/v4.0/Ch12/Ch12.md#defining-mapreduce-indexes-via-client-code"
+                }
+            ]
+        },
+        {
+            "Title": "Define the Reduce function",
+            "Slug" : "step-4",
+            "DescriptionLines": [
+                "* The `reduce` function is the second stage.<br>",
+                "  It operates on the Map results, specifying how the data should be _grouped_ and _aggregated_.",
+                "",
+                "* Specify the field to group-by:<br><br>",
+                "  `GroupBy(result => result.Country)`  ==> group by _index-field_ 'Country'",
+                "",
+                "* Specify the aggregation needed on that field:<br><br>",
+                "  `Country = g.Key`  ==>  The country we group by<br>",
+                "  `CountryCount = Enumerable.Sum(g, x => ((int) x.CountryCount))`  ==> The aggregating function"
```

Comment:
can we check if this cast to int is needed? less is better. :)

Reply:
not needed, works w/o